src/tests/ftest/soak/soak.py

```diff
@@ -784,8 +806,8 @@ def execute_jobs(self, jobs, pools):
         self.test_log_dir = self.log_dir + "/pass" + str(self.loop)
         self.local_pass_dir = self.outputsoakdir + "/pass" + str(self.loop)
         result = slurm_utils.srun(
-            NodeSet.fromlist(self.hostlist_clients), "mkdir -p {}".format(
-                self.test_log_dir), self.srun_params)
+            NodeSet.fromlist(self.hostlist_clients),
+            ["mkdir", "-p", self.test_log_dir], self.srun_params)
```

Comment:
Do we need a full path for the `mkdir` command?

Reply:
Not needed for mkdir